acme_diags/driver/lat_lon_vector_driver.py

```diff
@@ -0,0 +1,248 @@
+        MERIDIONAL = 1
+        var_title = '{}-{}'.format(parameter.variables[ZONAL], parameter.variables[MERIDIONAL])
+        for var in variables:
+            print('Variable: {}'.format(var))
+            parameter.var_id = var
+
+            mv1 = test_data.get_climo_variable(var, season)
+            mv2 = ref_data.get_climo_variable(var, season)
+
+            parameter.viewer_descr[var] = mv1.long_name if hasattr(
+                mv1, 'long_name') else 'No long_name attr in test data.'
+            s = parameter.viewer_descr[var].replace('Zonal ', '').replace('Meridional ', '').capitalize()
+            parameter.viewer_descr[var] = s
+            print('DEBUG: driver: parameter.viewer_descr[{}]={}'.format(var, parameter.viewer_descr[var]))
+
+
+            # For variables with a z-axis.
+            if mv1.getLevel() and mv2.getLevel():
+                plev = parameter.plevs
+                print('Selected pressure level: {}'.format(plev))
+
+                mv1_p = utils.general.convert_to_pressure_levels(mv1, plev, test_data, var, season)
+                mv2_p = utils.general.convert_to_pressure_levels(mv2, plev, ref_data, var, season)
+
+                # Select plev.
+                for ilev in range(len(plev)):
+                    mv1 = mv1_p[ilev, ]
+                    mv2 = mv2_p[ilev, ]
+
+                    for region in regions:
+                        print("Selected region: {}".format(region))
+
+                        mv1_domain = utils.general.select_region(region, mv1, land_frac, ocean_frac, parameter)
+                        mv2_domain = utils.general.select_region(region, mv2, land_frac, ocean_frac, parameter)
+
+                        parameter.output_file = '-'.join(
+                            [ref_name, var_title, str(int(plev[ilev])), season, region])
+                        #parameter.main_title = str(
+                        #    ' '.join([var, str(int(plev[ilev])), 'mb', season, region]))
+
+                        if var.find('TAU') == 0:
+                            parameter.main_title = str(' '.join(['TAU', str(int(plev[ilev])), 'mb', season, region]))
+                        elif var.find('U') ==0 or var.find('V') ==0:
+                            parameter.main_title = str(' '.join(['Wind', str(int(plev[ilev])), 'mb', season, region]))
+                        else:
+                            parameter.main_title = str(' '.join([var, str(int(plev[ilev])), 'mb', season, region]))
+    
+                        # Regrid towards the lower resolution of the two
+                        # variables for calculating the difference.
+                        mv1_reg, mv2_reg = utils.general.regrid_to_lower_res(
+                            mv1_domain, mv2_domain, parameter.regrid_tool, parameter.regrid_method)
+
+                        tests.append(mv1_reg)
+                        refs.append(mv2_reg)
+
+                        # TODO: Do we want to save these json files? Net CDF files?
```

Comment:
Should we save any json or Net CDF files?

Reply:
Yes, if save_netcdf is on, we should save at lease wind speed as netcdf. And the metrics based on wind speed should be saved following other lat-lon variables.